src/main/java/step4/Cars.java

```diff
@@ -0,0 +1,27 @@
+package step4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void printLocations(MaxLocation maxLocation) {
+        for (Car car : cars) {
+            System.out.print(car.getResult(maxLocation));
+        }
+        System.out.println();
+    }
+
+    public List<String> findWinnerName(MaxLocation maxLocation) {
+        List<String> carNameList = new ArrayList<>();
```

Comment:
```suggestion
        List<String> carNames = new ArrayList<>();
```
변수명에 자료구조를 포함시킬 경우, 자료구조가 변경되면 변수명도 변경해야할수도 있습니다